### PR TITLE
Add things to do in Anup's hometown 

### PR DESCRIPTION
**Pull request by asharma-art**
Added some text

### CODE REVIEW

File: anup-ktm.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#anup - ktm
```
Comment (senthil33) [inline anup-ktm.md:1]: @anup, Can you add town name and few details

**Review by senthil33 — CHANGES_REQUESTED** [review]
Looks good . Few missing pieces though

File: anup-ktm.md (hunk 1 above)
Comment (asharma-art) [reply anup-ktm.md:1]: Durbar sqaure;
Pashupati

**Review by senthil33 — APPROVED** [review]

**Comment by asharma-art** [comment]
Merging to master

**Review by aramaniart — APPROVED** [review]

**Review by aramaniart — APPROVED** [review]

**Pull request merged**